homeassistant/components/ruckus_unleashed/coordinator.py

```diff
@@ -37,12 +31,19 @@ def __init__(self, hass: HomeAssistant, *, ruckus: Ruckus) -> None:
 
     async def _fetch_clients(self) -> dict:
         """Fetch clients from the API and format them."""
-        clients = await self.ruckus.current_active_clients()
-        return {e[API_MAC]: e for e in clients[API_CURRENT_ACTIVE_CLIENTS][API_CLIENTS]}
+        try:
+            clients = await self.ruckus.api.get_active_clients()
+        except Exception as error:
```

Comment:
We don't allow catching the bare Exception (except in the config_flow). Please refactor the code to catch only exception fired by the library.